packages/utils/nft.ts

```diff
@@ -88,4 +90,18 @@ export const nftCardInfoFromStargazeIndexerNft = (
   imageUrl: token.media?.visualAssets?.lg?.url || token.media?.url || undefined,
   name: token.name || token.tokenId || 'Unknown NFT',
   description: token.description || undefined,
+  highestOffer:
+    token.highestOffer && token.highestOffer.offerPrice
+      ? {
+          amount:
+            token.highestOffer.offerPrice.amount &&
+            convertMicroDenomToDenomWithDecimals(
+              token.highestOffer.offerPrice.amount,
+              6
+            ),
+          amountUsd: token.highestOffer.offerPrice.amountUsd,
+          denom: 'STARS',
+        }
+      : undefined,
```

Comment:
`offerPrice` seems to support different denoms, so we can't assume that this is a STARS token with 6 decimals. instead, i think this helper function should take a `token: GenericToken` argument, which you can retrieve in the selector that uses this helper function with `get(genericTokenSelector(...))`

also instead of doing any conversion here, `highestOffer` should contain that `token` field directly so it can be referenced wherever necessary, and perform the amount decimal conversions right in the component.